pageserver/src/http/openapi_spec.yml

```diff
@@ -909,35 +909,27 @@ components:
             See the tenant `/attach` endpoint for more information.
           type: string
           enum: [ "maybe", "attached" ]
-    TenantCreateInfo:
-      type: object
-      properties:
-        new_tenant_id:
-          type: string
-          format: hex
-        tenant_id:
-          type: string
-          format: hex
-        gc_period:
-          type: string
-        gc_horizon:
-          type: integer
-        pitr_interval:
-          type: string
-        checkpoint_distance:
-          type: integer
-        checkpoint_timeout:
-          type: string
-        compaction_period:
-          type: string
-        compaction_threshold:
-          type: string
-    TenantConfigInfo:
+    TenantCreateRequest:
+      allOf:
```

Comment:
@problame 
so this the magic ? it will take the TenantConfig and extract all of its members? so the current clients need not to change ?

Reply:
Yup, see corresponding https://github.com/neondatabase/cloud/pull/5046

Reply:
nice !